src/core/initial_conditions.jl

```diff
@@ -4,6 +4,11 @@ Initial condition for Upper reservoir in [`PowerSystems.HydroPumpedStorage`](@ex
 struct InitialHydroEnergyLevelUp <: PSI.InitialConditionType end
 
 """
-Initial condition for Down reservoir in [`PowerSystems.HydroPumpedStorage`](@extref) formulations
+Initial condition for Down reservoir in [`PowerSystems.HydroEnergyBlockOptimization`](@extref) formulations
```

Comment:
This docstring is wrong. I'm guessing you want to say `in [`PowerSystems.HydroPumpedStorage`](@extref) or in [`PowerSystems.HydroReservoir`](@extref) formulations `